k8s/mainflux/ws.yml

```diff
@@ -45,7 +45,7 @@ spec:
     apiVersion: extensions/v1beta1
     kind: Deployment
     name: ws-adapter
-  minReplicas: 3
+  minReplicas: 1
```

Comment:
I'm not sure that we want to set min replicas as 1.

Reply:
I turned it back to 3 since I used 1 for my tests (I don't have a big system), if you think this number must be changed, I can put whatever you think we should have there.